Shop/MainWindow.xaml

```diff
@@ -0,0 +1,57 @@
+﻿<Window x:Class="Shop.MainWindow"
+        xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+        xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+        xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+        xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+        xmlns:local="clr-namespace:Shop"
+        xmlns:model="clr-namespace:Shop.Model"
+        xmlns:view="clr-namespace:Shop.View"
+        xmlns:viewModel="clr-namespace:Shop.ViewModel"
+        mc:Ignorable="d"
+        Title="MainWindow" Height="350" Width="800">
+    <Window.DataContext>
+        <viewModel:CarListViewModel/>
+    </Window.DataContext>
+    <Window.Resources>
+        <DataTemplate DataType="{x:Type model:Car}">
+            <Grid>
```

Comment:
I'd suggest to make layout a littile bit user-friendly - add margins and paddings.

Reply:
I'd suggest to move this DataTemplate tags to a separate control.